README.md

```diff
@@ -33,7 +35,142 @@ An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
+
+```sh
+npm db:setup
+```
+
+## Server API
+
+### Get place info
+  * GET `/api/places/:placeId`
+
+**Path Parameters:**
+  * `placeId` place id
+
+**Success Status Code:** `200`
+
+**Returns:** JSON
+
+```json
+    {
+        "placeId": "Number",
+        "nightlyFee": "Number",
+        "cleaningFee": "Number",
+        "occupancyTaxRate": "Number",
+        "averateRating": "Number",
+        "reviews": "Number",
+        "locationCity": "String",
+        "locationCountry": "String",
+        "maxCapacity": "Number",
+        "bookings": [
+          {
+            "bookingId": "Number",
+            "checkin": "Date",
+            "checkout": "Date",
+          },
+          {
+            "bookingId": "Number",
+            "checkin": "Date",
+            "checkout": "Date",
+          },
+        ]
+    },
+```
+
+### Get booking info
+  * GET `/api/bookings/:bookingId`
+
+**Path Parameters:**
+  * `bookingId` place id
+
+**Success Status Code:** `200`
+
+**Returns:** JSON
+
+```json
+    {
+      "bookingId": "Number",
+      "checkin": "Date",
+      "checkout": "Date",
+      "maxCapacity": "Number",
+      "adults": "Number",
+      "children": "Number",
+      "infants": "Number",
+      "placeId": "Number",
+      "nightlyFee": "Number",
+      "cleaningFee": "Number",
+      "occupancyTaxRate": "Number",
+      "locationCity": "String",
+      "locationCountry": "String",
+    },
 ```
 
+### Add a new booking to existing place
+  * POST `/api/places/:placeId`
+
+**Path Parameters:**
+  * `placeId` place id
+
+**Success Status Code:** `201`
+
+**Request Body**: Expects JSON with the following keys to add a new booking to existing place
+
+```json
+    {
+      "placeId": "Number",
+      "nightlyFee": "Number",
+      "cleaningFee": "Number",
+      "occupancyTaxRate": "Number",
+      "checkin": "Date",
+      "checkout": "Date",
+      "adults": "Number",
+      "children": "Number",
+      "infants": "Number",
+    }
+```
+
+### Update the existing booking info
+  * PATCH `/api/booking/:bookingId`
+
+**Path Parameters:**
+  * `bookingId` booking id
+
+**Success Status Code:** `204`
+
+**Request Body**: Expects JSON with following keys to update the booking info of existing booking. Checkin, checkout, adults, children, infants fields are required. Other fields are optional because the booking can use existing nightlyFee, cleaningFee, occupancyTaxRate recalculate booking total.
+
+```json
+    {
+      "bookingId": "Number",
+      "checkin": "Date",
+      "checkout": "Date",
+      "adults": "Number",
+      "children": "Number",
+      "infants": "Number",
+      "nightlyFee": "Number",
+      "cleaningFee": "Number",
+      "occupancyTaxRate": "Number",
+    }
+```
+
+### Delete a place from database
+  * DELETE `/api/places/:placeId`
+
+**Path Parameters:**
+  * `placeId` place id
+
+**Success Status Code:** `204`
+
+### Delete a booking from database
+  * DELETE `/api/bookings/bookingId`
```

Comment:
I think you need a colon in front of :bookingId. Otherwise this looks great Tegshee. Makes me realize I might need to fill mine out a bit more :sweat_smile:

Reply:
nice catch. fixed it.